rustls-mbedcrypto-provider/src/tls13.rs

```diff
@@ -191,3 +193,123 @@ impl MessageDecrypter for Tls13MessageDecrypter {
         msg.into_tls13_unpadded_message()
     }
 }
+
+struct MbedHkdfUsingHmac<'a>(&'a super::hmac::Hmac);
+
+const ZERO_IKM: [u8; crate::hmac::Tag::MAX_LEN] = [0u8; crate::hmac::Tag::MAX_LEN];
+
+impl<'a> Hkdf for MbedHkdfUsingHmac<'a> {
+    fn extract_from_zero_ikm(&self, salt: Option<&[u8]>) -> Box<dyn HkdfExpander> {
+        let md = self.0.hash_algorithm().hash_type;
+        let capacity = self.0.hash_algorithm().output_len;
+        let mut prf = crate::hmac::Tag::with_capacity(capacity);
+        let _ = mbedtls::hash::Hkdf::hkdf_extract(md, salt, &ZERO_IKM[..capacity], prf.as_mut())
+            .map_err(|_err| error!("MbedHkdf::extract_from_zero_ikm got mbedtls error: {:?}", _err));
```

Comment:
Can we Send the error to our `HkdfExpander` and return it from there? I don't think ignoring the error is ideal. 
If you think we should never, ever see these errors, I'd be OK with panicking here instead of ignoring the error.

Reply:
Make sense, will change it